agent/xds/clusters_test.go

```diff
@@ -540,12 +550,22 @@ func TestClustersFromSnapshot(t *testing.T) {
 				return proxycfg.TestConfigSnapshotIngressGateway(t, true, "tcp",
 					"simple", nil,
 					func(entry *structs.IngressGatewayConfigEntry) {
+						defaultEnforcingConsecutive5xx := uint32(80)
 						entry.Defaults = &structs.IngressServiceConfig{
 							MaxConnections:     2048,
 							MaxPendingRequests: 512,
+							PassiveHealthCheck: &structs.PassiveHealthCheck{
+								Interval:                5000000000,
+								EnforcingConsecutive5xx: &defaultEnforcingConsecutive5xx,
+							},
 						}
+						enforcingConsecutive5xx := uint32(50)
 						entry.Listeners[0].Services[0].MaxConnections = 4096
 						entry.Listeners[0].Services[0].MaxPendingRequests = 2048
+						entry.Listeners[0].Services[0].PassiveHealthCheck = &structs.PassiveHealthCheck{
```

Comment:
It might be safer to add the tests for outlier detection to new cases so it's easier to ensure the existing cases don't change.